src/sched/lthread.c

```diff
@@ -27,6 +27,8 @@
  *
  */
 
+// clang-format off
```

Comment:
I don’t see the point of keeping different formatting here, as we are not tracking any upstream.

Reply:
OK makes sense, the license header mislead me.